app/models/proxy_deposit_request.rb

```diff
@@ -75,6 +75,10 @@ def cancel!
     save!
   end
 
+  def is_canceled?
+    self.status == 'canceled'
```

Comment:
Let's make `'canceled'` a constant (and use it in `cancel!`) too.  Also, can you add a test to this PR?